pkg/controller/podautoscaler/horizontal.go

```diff
@@ -56,8 +56,15 @@ var (
 	scaleUpLimitMinimum = 4.0
 )
 
+type specReplicas int32
```

Comment:
Can you leave a comment on why this distinction is important?

Reply:
Also, is there a way to distinguish the types in a more meaningful way? i.e. observedReplicas vs. desiredReplicas?
I'm not sure that it will be clearer, but want to hear your thoughts on this.